### ISSUE TITLE
Hindi locale

### ISSUE THREAD

**Issue by amyjko** [wordplaydev/wordplay#140]
**What's the problem?**
We need a Hindi locale.

**Who benefits?**
All fluent Hindi readers; ~7% of the world. Use machine translation to start, then edit.

**Comment by yshastri5** [comment]
@amyjko I would like to work on issue. it would be amazing if you can add me to the issue.

**Comment by adhimanjagota** [comment]
Hi Amy, 
I'm quite fluent in Hindi and would like to resolve this issue. Can you please assign it to me?

**Comment by adhimanjagota** [comment]
Hey! I've created the folder to resolve the issue. Here's the location - wordplay/static/locales/main/hn-IN

**Comment by amyjko** [comment]
Added you both!

**Comment by adhimanjagota** [comment]
Hey! We have completed the localisation. Should I just upload it to the folder?

**Comment by amyjko** [comment]
@adhimanjagota All contributions should be submitted as pull requests, per the workflow defined in the wiki. Read the localization wiki for instructions.

**Comment by amyjko** [comment]
Partially resolved by PR #622.

**Comment by tris-23** [comment]
hi @amyjko can I get added to this issue? I would like to recheck some of the translations.

**Comment by ananyat24** [comment]
Hi @amyjko! Can I get added to this issue as well? Thank you!

**Comment by amyjko** [comment]
Added you both.

@adhimanjagota @yshastri5, are you still contributing to this? Please let us know one way or another, so we don't have people doing duplicate work.

**Comment by adhimanjagota** [comment]
@amyjko yes we're still contributing to this issue and will love to add a value to the community. We'll be attending the meet up next week to divide up the work with @ananyat24

**Comment by amyjko** [comment]
Thanks @adhimanjagota! Coordination is key; I recommend working in a shared branch on this, so you don't do redundant work and can identify conflicts early.

**Comment by daminihs** [comment]
Hi @adhimanjagota and team working on this, I reviewed the Hindi JSON file and noticed a couple of discrepancies. I’m not sure if work has already started on these or if there’s a specific reason they’re structured this way, but I thought I thought I’d check with you all.
1.  "Stream" translation in 'term' is different than 'token '- token has the correct translation (dhara).
2. "Language" is  translated to "bhasha tag" - not sure if the ~tag~ was intentional
3. "TypeOperator" is just "hai" - incorrect
4. "table" is translated to a dining table "mez". should be something like तालिका . and should be consistent with "TableOpen" / "TableClose"
5. "set" in term is incorrect
6. "entered" in term is incorrect

**Comment by amyjko** [comment]
Assigned! And unassigning @adhimanjagota @tris-23 and @yshastri5 who ghosted.

**Comment by amyjko** [comment]
Hi! I'm Amy's GitHub bot. @ellipsis7449 — Are you still working on this issue? Can you post a status update, and if you won't finish it, can you unassign yourself? We'll unassign you in a week if we don't get a reply.

**Comment by ellipsis7449** [comment]
Planning to work on this in the summer!

**Comment by amyjko** [comment]
Hi! I'm Amy's GitHub bot. @ellipsis7449 — Are you still working on this issue? Can you post a status update, and if you won't finish it, can you unassign yourself? We'll unassign you in a week if we don't get a reply.

**Comment by ellipsis7449** [comment]
Yes, I'm going to hop on this soon, after I've reviewed Adrienne's Wordplay unit plan. Will check out the new machine translations :)

**Comment by amyjko** [comment]
Hi! I'm Amy's GitHub bot. @ellipsis7449 — Are you still working on this issue? Can you post a status update, and if you won't finish it, can you unassign yourself? We'll unassign you in a week if we don't get a reply.

**Comment by ellipsis7449** [comment]
Getting back onto this 👍

**Comment by amyjko** [comment]
Yay!

**Comment by amyjko** [comment]
Hi! I'm Amy's GitHub bot. @ellipsis7449 — Are you still working on this issue? Can you post a status update, and if you won't finish it, can you unassign yourself? We'll unassign you in a week if we don't get a reply.

**Comment by amyjko** [comment]
Hi! I'm Amy's GitHub bot. @ellipsis7449 — Are you still working on this issue? Can you post a status update, and if you won't finish it, can you unassign yourself? We'll unassign you in a week if we don't get a reply.

**Comment by ellipsis7449** [comment]
Yes, I am. Just been working on collecting terms from the Hindi math textbook and seeing if/how I can incorporate them into the locale text.

**Comment by amyjko** [comment]
Hi! I'm Amy's GitHub bot. @ellipsis7449 — Are you still working on this issue? Can you post a status update, and if you won't finish it, can you unassign yourself? We'll unassign you in a week if we don't get a reply.

**Comment by ellipsis7449** [comment]
Not planning to work on it for the rest of summer, but will probably get back into it once the school year starts.

**Comment by amyjko** [comment]
Sounds good, thanks @ellipsis7449.
